[Docs] Add link to dependencies page - 8.x (#214667)
elastic/kibana: docs/developer/dependencies-versions.asciidoc

[[dependencies-versions]]
== List of dependencies

> Refer to https://artifacts.elastic.co/reports/dependencies/dependencies-current.html[Elastic Stack third-party dependencies] for a list of the external dependencies you can leverage.

Reviewer: ```suggestion
Refer to https://artifacts.elastic.co/reports/dependencies/dependencies-current.html[Elastic Stack third-party dependencies] for the complete list of dependencies.
```
